lib/Config.lua

```diff
@@ -18,6 +18,8 @@ local defaultConfig = {
 	["elementTracing"] = false,
 	-- Enables instrumentation of shouldUpdate and render methods for Roact components
 	["componentInstrumentation"] = false,
+	-- Enables type checking with the validateProps property on stateful components.
+	["propertyValidation"] = false,
```

Comment:
Much like the recommendation in the docs, let's call this one `propValidation` to be more exact.